modules/virt-viewing-automatically-created-storage-profiles.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-viewing-automatically-created-storage-profiles_{context}"]
+= Viewing automatically created storage profiles
+
+The system creates storage profiles for each storage class automatically.
+
+. To view the list of storage profiles, run the following command:
++
+[source,terminal]
+----
+oc get storageprofile
+----
+
+. To fetch the details of a particular storage profile, run the following command:
++
+[source,terminal]
+----
+oc describe storageprofile <name>
+----
+
+.Example storage profile details
+[source,yaml]
+----
+$ oc describe  storageprofile ocs-storagecluster-ceph-rbd-virtualization
+Name:         ocs-storagecluster-ceph-rbd-virtualization
+Namespace:
+Labels:       app=containerized-data-importer
+              app.kubernetes.io/component=storage
+              app.kubernetes.io/managed-by=cdi-controller
+              app.kubernetes.io/part-of=hyperconverged-cluster
+              app.kubernetes.io/version=4.17.2
+              cdi.kubevirt.io=
+Annotations:  <none>
+API Version:  cdi.kubevirt.io/v1beta1
+Kind:         StorageProfile
+Metadata:
+  Creation Timestamp:  2023-11-13T07:58:02Z
+  Generation:          2
+  Owner References:
+    API Version:           cdi.kubevirt.io/v1beta1
+    Block Owner Deletion:  true
+    Controller:            true
+    Kind:                  CDI
+    Name:                  cdi-kubevirt-hyperconverged
+    UID:                   2d6f169a-382c-4caf-b614-a640f2ef8abb
+  Resource Version:        4186799537
+  UID:                     14aef804-6688-4f2e-986b-0297fd3aaa68
+Spec:
+Status:
+  Claim Property Sets: <1>
+    accessModes:
+      ReadWriteMany
+    volumeMode:  Block
+    accessModes:
+      ReadWriteOnce
+    volumeMode:  Block
+    accessModes:
+      ReadWriteOnce
+    volumeMode:                   Filesystem
+  Clone Strategy:                  csi-clone <2>
+  Data Import Cron Source Format:  snapshot <3>
+  Provisioner:                     openshift-storage.rbd.csi.ceph.com
+  Snapshot Class:                  ocs-storagecluster-rbdplugin-snapclass
+  Storage Class:                   ocs-storagecluster-ceph-rbd-virtualization
+Events:                            <none>
+----
+<1> `Claim Property Sets` is an ordered list of `AccessMode`/`VolumeMode` pairs, which describe the PVC modes that will be used to provision VM disks.
```

Comment:
In general, stay away from "future tense": https://spaces.redhat.com/display/CCSENABLE/Future+tense

```suggestion
<1> `Claim Property Sets` is an ordered list of `AccessMode`/`VolumeMode` pairs, which describe the PVC modes that are used to provision VM disks.
```